src/Grpc.AspNetCore.Server/Internal/GrpcServerLog.cs

```diff
@@ -21,229 +21,89 @@
 
 namespace Grpc.AspNetCore.Server.Internal;
 
-internal static class GrpcServerLog
+internal static partial class GrpcServerLog
 {
-    private static readonly Action<ILogger, Exception?> _unableToDisableMaxRequestBodySize =
-        LoggerMessage.Define(LogLevel.Debug, new EventId(1, "UnableToDisableMaxRequestBodySizeLimit"), "Unable to disable the max request body size limit.");
+    [LoggerMessage(Level = LogLevel.Trace, EventId = 27, EventName = "DeadlineStopped", Message = "Request deadline stopped.")]
```

Comment:
Could you preserve the previous order? I'd like to have them ordered by the event id. That way new log methods can be added to the end and it's easy to see the previous event id.

Also, keeping that order also makes code reviewing the changes easier.